.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,27 @@
+<!--
+Thanks for contributing to XOOPS! Fill in the sections below.
+The checklist encodes what the review bots and maintainers check on
+every PR - working through it honestly shortens your review cycle.
+-->
+
+### Why
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)